app/models/sales_log.rb

```diff
@@ -379,12 +383,13 @@ def beds_for_la_sale_range
     beds.nil? ? nil : [beds, LaSaleRange::MAX_BEDS].min
   end
 
-  def ownership_scheme
-    case ownershipsch
-    when 1 then "shared ownership"
-    when 2 then "discounted ownership"
-    when 3 then "outright sale"
-    end
+  def ownership_scheme(uppercase: false)
+    owner_scheme = case ownershipsch
```

Comment:
can this variable be called ownership_scheme please or just scheme or scheme_title if that causes some kind of naming conflict with the method itself

Reply:
I had to reread that 4 times to spot it on 387, yes good spot :)